### ISSUE TITLE
Importing the entire world with nominatim-docker

### ISSUE THREAD

**Issue by aviNrich** [mediagis/nominatim-docker#261]
Hi.
Is it possible to import the entire world to the database? 
If so, can you share the correct docker commands?

Thanks

**Comment by leonardehrenfried** [comment]
Yes, it's possible but will require a machine with 64GB of memory: https://nominatim.org/release-docs/latest/admin/Installation/#hardware

The original planet file from https://planet.openstreetmap.org/ is throttled so you want to use a mirror from: https://wiki.openstreetmap.org/wiki/Planet.osm#Planet.osm_mirrors

Choose a mirror near your server for the best download speed.

If you're in Europe you could use the German mirror. I haven't tested it myself but i think it's this command:

```
docker run -it \
  -e PBF_URL=https://ftp5.gwdg.de/pub/misc/openstreetmap/planet.openstreetmap.org/pbf/planet-latest.osm.pbf \
  -e REPLICATION_URL=https://ftp5.gwdg.de/pub/misc/openstreetmap/planet.openstreetmap.org/replication/day/ \
  -p 8080:8080 \
  --name nominatim \
  mediagis/nominatim:3.7
```

You might need to also play around with the tuning parameters documented in the wiki.

Please report back if it works.

**Comment by aviNrich** [comment]
Thank you.
1. Can you please estimate the load time 4 cores + 64GB?
2. Once imported, can I reduce the memory?

Thanks

**Comment by mtmail** [comment]
Prepare for 5-10 days of importing. It's easier to estimate once the first couple of ranks are imported.

Disc speed is also a factor, use the fastest discs (or network mounted storage) you can find.

After the import you can lower the RAM.
